### ISSUE TITLE
How to setup own DNS on localhost ?

### ISSUE THREAD

**Issue by thepenguinthatwants** [PARC6502/docker-media-server#1]
Hi man!

Your setup looks awesome. But just when I was starting to test it up I feel like even getting it up seems problematic. 

Where is the content folder? IT was mentioned in your readme.md but it wasnt on the mkdir -p? Do you mean those DLDIR TVDIR MOVIESDIR?

The main part that felt new to me was.
Setup your DNS so that all *.yourdomain go to the media server. Your domain doesn't have to be registered since it will only be used internally

How do you do that? Editing dnsmasq or host files? :D tho I guess u r probably going to update this part later on.

**Comment by PARC6502** [comment]
> Where is the content folder? IT was mentioned in your readme.md but it wasnt on the mkdir -p? Do you mean those DLDIR TVDIR MOVIESDIR?

Yeah, that's right. I didn't want to put them in the command explicitly in case people wanted their contents somewhere different

> How do you do that? Editing dnsmasq or host files?

I didn't put details in this repo since the specifics will vary depending on your setup. I personally use dnsmasq inside a docker container [as shown here.](https://github.com/PARC6502/docker-dns-dhcp) I will try to update that repo with more info about that particular setup. Otherwise you may have a router that has DNS functionality. Even if your router doesn't and only has full DHCP (this is what automatically assigns IPs, and also provides other information like the IP address of the router and the IP address of the DNS server) capability, that would simplify the setup.

**Comment by thepenguinthatwants** [comment]
Hi!

Thanks for the reply.

I am wondering if the docker-dns-dhcp is needed on localhost one laptop setup?

So I am aiming to make it work only locally but in a way its okay if its open on my network as I am using wifi.  There is example.env that needs to be changed into .env    Do I need to define DOMAIN=example.com into localhost? or something if I want to use subdomains locally?

**Comment by thepenguinthatwants** [comment]
Hi thanks for the reply.

I feel cool enough that I can access the images with localhost:9090/api/rawdata

Looking at docker-dns-dhcp repo its like vast of things that I have no clue of.

**Comment by PARC6502** [comment]
Let me know what you're struggling with and I'll try to help you out
